### ISSUE TITLE
presence of SDL2 not checked?

### ISSUE THREAD

**Issue by mhinsch** [jonathanBieler/SimpleDirectMediaLayer.jl#21]
It appears the package happily installs if only libSDL but not libSDL2 is installed, but then crashes on .init().

[Ubuntu 16.4, julia 0.7 beta 2]

**Comment by jonathanBieler** [comment]
Maybe it's the libSDL alias here:

https://github.com/jonathanBieler/SimpleDirectMediaLayer.jl/blob/master/deps/build.jl#L6

Can you check what the proper name of the library on your system ? (apt-cache search sdl)

**Comment by mhinsch** [comment]
I have libsdl1.2debian:amd64 and libsdl1.2-dev (and various sdl-* packages of course) installed.
I vaguely remember having had a similar issue with a CMAKE based project a while ago where we ended up concluding that SDL versioning has some serious issues.

**Comment by jonathanBieler** [comment]
I mean the name of the one that should be installed, we basically need to make sure the version 2 get installed instead of the 1, but for that we need the correct name that your package manager expect.

**Comment by mhinsch** [comment]
Ok, I'm not entirely sure which information you need, so here's the full dump:

```
dpkg -l *sdl*
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name                                 Version                 Architecture            Description
+++-====================================-=======================-=======================-=============================================================================
ii  freeciv-client-sdl                   2.5.3-1                 amd64                   Civilization turn based strategy game (SDL client)
un  libsdl-1.3-0                         <none>                  <none>                  (no description available)
un  libsdl-1.3-dev                       <none>                  <none>                  (no description available)
un  libsdl-dev                           <none>                  <none>                  (no description available)
rc  libsdl-gfx1.2-4:amd64                2.0.23-3                amd64                   drawing and graphical effects extension for SDL
ii  libsdl-gfx1.2-5:amd64                2.0.25-5                amd64                   drawing and graphical effects extension for SDL
ii  libsdl-image1.2:amd64                1.2.12-5+deb9u1build0.1 amd64                   Image loading library for Simple DirectMedia Layer 1.2, libraries
ii  libsdl-mixer1.2:amd64                1.2.12-11build1         amd64                   Mixer library for Simple DirectMedia Layer 1.2, libraries
ii  libsdl-net1.2:amd64                  1.2.8-4                 amd64                   Network library for Simple DirectMedia Layer 1.2, libraries
ii  libsdl-sound1.2:amd64                1.0.3-7build1           amd64                   Sound library for Simple DirectMedia Layer 1.2, libraries
ii  libsdl-ttf2.0-0:amd64                2.0.11-3                amd64                   TrueType Font library for Simple DirectMedia Layer 1.2, libraries
un  libsdl1.2                            <none>                  <none>                  (no description available)
un  libsdl1.2-all                        <none>                  <none>                  (no description available)
ii  libsdl1.2-dev                        1.2.15+dfsg1-3          amd64                   Simple DirectMedia Layer development files
un  libsdl1.2-esd                        <none>                  <none>                  (no description available)
un  libsdl1.2-nas                        <none>                  <none>                  (no description available)
un  libsdl1.2-oss                        <none>                  <none>                  (no description available)
ii  libsdl1.2debian:amd64                1.2.15+dfsg1-3          amd64                   Simple DirectMedia Layer
un  libsdl1.2debian-all                  <none>                  <none>                  (no description available)
un  libsdl1.2debian-alsa                 <none>                  <none>                  (no description available)
un  libsdl1.2debian-esd                  <none>                  <none>                  (no description available)
un  libsdl1.2debian-nas                  <none>                  <none>                  (no description available)
un  libsdl1.2debian-oss                  <none>                  <none>                  (no description available)
un  libsdl1.2debian-pulseaudio           <none>                  <none>                  (no description available)
ii  libsdl2-2.0-0:amd64                  2.0.4+dfsg1-2ubuntu2.16 amd64                   Simple DirectMedia Layer
ii  libsdl2-dev                          2.0.4+dfsg1-2ubuntu2.16 amd64                   Simple DirectMedia Layer development files
ii  libsdl2-mixer-2.0-0:amd64            2.0.1+dfsg1-1           amd64                   Mixer library for Simple DirectMedia Layer 2, libraries
ii  libsdl2-mixer-dev:amd64              2.0.1+dfsg1-1           amd64                   Mixer library for Simple DirectMedia Layer 2, development files
ii  libsdl2-ttf-2.0-0:amd64              2.0.14+dfsg1-1          amd64                   TrueType Font library for Simple DirectMedia Layer 2, libraries
ii  libsdl2-ttf-dev:amd64                2.0.14+dfsg1-1          amd64                   TrueType Font library for Simple DirectMedia Layer 2, development files
```

**Comment by giordano** [comment]
Fixed by #41?
